armory/__main__.py

```diff
@@ -268,10 +268,19 @@ def run(command_args, prog, description):
         action="store_true",
         help="Skip benign inference and metric calculations",
     )
+    parser.add_argument(
+        "--skip-attack",
+        action="store_true",
+        help="Skip attack generation and metric calculations",
+    )
 
     args = parser.parse_args(command_args)
     coloredlogs.install(level=args.log_level)
 
+    if args.skip_benign and args.skip_attack:
+        logger.error("--skip-benign and --skip-attack cannot both be passed. ")
+        sys.exit(1)
+
```

Comment:
I would probably allow this to happen, particularly for the case if someone just wanted to train a model and ensure that piece worked, but it could be helpful as a test no-op.